wrappers/swift/SemanticContext.swift

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper over `ODWSemanticContext` class that manages the inclusion of semantic context values on logged events.
+public class SemanticContext {
```

Comment:
Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.

Reply:
It has instance methods so potentially might be some advantage to subclassing in future. keeping it non-final.